guides/configuring_mailer.md

```diff
@@ -0,0 +1,80 @@
+# Configuring mailer
+
+You are able to configure a mailer for Pow and set it up with the library of 
+your choice.
+
+This guide shows how to setup Pow with
+
+ * [Swoosh](https://github.com/swoosh/swoosh)
+ * [Bamboo](https://github.com/thoughtbot/bamboo)
+
+You must first setup and configure either of these libraries before you can
+integrate them with Pow.
+
+## Swoosh mailer
+
+Set up your `WEB_PATH/pow_mailer.ex` file like so:
+
+```elixir
+defmodule MyAppWeb.PowMailer do
+  use Pow.Phoenix.Mailer
+  use Swoosh.Mailer, otp_app: :my_app
+
+  import Swoosh.Email
+  
+  require Logger
+
+  def cast(%{user: user, subject: subject, text: text, html: html}) do
+    %Swoosh.Email{}
+    |> to({"", user.email})
+    |> from({"My App", "myapp@example.com"})
+    |> subject(subject)
+    |> html_body(html)
+    |> text_body(text)
+  end
+
+  def process(email) do
+    email
+    |> deliver()
+    |> log_warnings()
+  end
+
+  defp log_warnings({:error, reason}) do
+    Logger.warn("Mailer backend failed with: #{inspect(reason)}")
+  end
+
+  defp log_warnings({:ok, response}), do: {:ok, response}
+end
+```
+
+Remember to add `mailer_backend: MyAppWeb.PowMailer` to the Pow configuration.
+
+## Bamboo mailer
+
+Set up your `WEB_PATH/pow_mailer.ex` file like so:
+
+```elixir
+defmodule MyAppWeb.PowMailer do
+  use Pow.Phoenix.Mailer
+  use Bamboo.Mailer, otp_app: :my_app
+
+  import Bamboo.Email
+  
+  require Logger
```

Comment:
This isn't used in this snippet. I'm assuming that it isn't needed here?

Reply:
Good catch, you're right. I've [pushed a commit](https://github.com/danschultzer/pow/commit/e3f8a0638dd4ad32f61f89cf2522cc1bd81e6f5e to master to fix it, thanks.